src/OpenFeature/OpenFeature.csproj

```diff
@@ -7,6 +7,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsLoggerVer)" />
+    <PackageReference Include="System.Threading.Channels" Version="8.0.0" />
```

Comment:
Probably worth moving this to [Common.props](https://github.com/open-feature/dotnet-sdk/blob/main/build/Common.props#L27) and adding a range from [6.0.0,)

Reply:
Thank you for the suggestion, I have updated the PR accordingly :)